src/frontend/komponenter/Fagsak/Vedtak/OppsummeringVedtak.tsx

```diff
@@ -0,0 +1,53 @@
+import { Knapp } from 'nav-frontend-knapper';
+import { Systemtittel } from 'nav-frontend-typografi';
+import * as React from 'react';
+
+import { hentVedtaksbrev, IVedtaksBrev } from '../../../api/oppsummeringvedtak';
+import { IFagsak } from '../../../typer/fagsak';
+import { Ressurs, RessursStatus } from '../../../typer/ressurs';
+
+interface IVedtakProps {
+    fagsak: IFagsak;
+}
+
+interface IOppsummeringFrameProps {
+    content: string;
+}
+
+const OppsummeringFrame: React.FunctionComponent<IOppsummeringFrameProps> = ({ ...props }) => {
+    return <iframe className="iframe" srcDoc={props.content} />;
```

Comment:
Can we move this directly to the component? Since it is so simple I dont see the advantage of putting it in a seperate component.

Reply:
done